src/detect.h

```diff
@@ -298,13 +298,15 @@ typedef struct DetectPort_ {
 #define DETECT_ENGINE_THREAD_CTX_INSPECTING_STREAM 0x0002
 #define DETECT_ENGINE_THREAD_CTX_STREAM_CONTENT_MATCH 0x0004
 
-#define FILE_SIG_NEED_FILE          0x01
-#define FILE_SIG_NEED_FILENAME      0x02
-#define FILE_SIG_NEED_TYPE          0x04
-#define FILE_SIG_NEED_MAGIC         0x08    /**< need the start of the file */
-#define FILE_SIG_NEED_FILECONTENT   0x10
-#define FILE_SIG_NEED_MD5           0x20
-#define FILE_SIG_NEED_SIZE          0x40
+#define FILE_SIG_NEED_FILE          0x0001
+#define FILE_SIG_NEED_FILENAME      0x0002
+#define FILE_SIG_NEED_TYPE          0x0004
```

Comment:
looks like this is unused, so it can be removed. Then the 8 bits are enough again :)